src/com/griddynamics/main/Ticket.java

```diff
@@ -0,0 +1,53 @@
+package com.griddynamics.main;
+
+import java.util.Scanner;
+
+public class Ticket {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please enter: ");
+        String ticket = in.nextLine();
+        isMyTicketLucky(ticket);
+        in.close();
+    }
+
+    public static Answer isMyTicketLucky(String ticketNumber) {
+        if (ticketNumber.matches("\\d{6}")) {
+            char[] x = new char[3];
+            char[] y = new char[3];
+            ticketNumber.getChars(0, 3, x, 0);
+            ticketNumber.getChars(3, 6, y, 0);
+            int sumX = 0;
+            int sumY = 0;
+            for (int i = 0; i < 3; i++) {
+                sumX += x[i];
+                sumY += y[i];
+            }
+            if (sumX == sumY) {
+                System.out.println("Congratulations! You have a lucky ticket!");
+                return Answer.LUCKY;
+            } else {
+                System.out.println("Sorry, try again later.");
+                return Answer.NOT_LUCKY;
+            }
+        } else {
+            System.out.println("Give correct ticket, please.");
+            return Answer.INVALID_TICKET;
```

Comment:
@kolegova-mary, I would suggest to use some kind of exception instead of INVALID_TICKET enum, for example IllegalArgumentException

Reply:
Done